File: OPENAPI-REST-API/openapi-client/php-dt/src/App/DTO/IpLimitRange.php
<?php
declare(strict_types=1);

namespace App\DTO;

use Articus\DataTransfer\PhpAttribute as DTA;

/**
 * The lower and upper bounds of an ip range.
 */
class IpLimitRange
{
    /**
     * The begining (or first) IP address in the range.
     */
    #[DTA\Data(field: "start")]
    #[DTA\Validator("Scalar", ["type" => "string"])]
    public string|null $start = null;

    /**
     * The ending (or last) IP address in the range.
     */
    #[DTA\Data(field: "end")]
    #[DTA\Validator("Scalar", ["type" => "string"])]
    public string|null $end = null;

}

File: OPENAPI-REST-API/swagger-client/scala-akka-http-server/src/main/scala/io/swagger/server/model/IpLimitRange.scala
package io.swagger.server.model


/**
 * = IpLimitRange =
 *
 * The lower and upper bounds of an ip range.
 *
 * @param start The begining (or first) IP address in the range. for example: ''1.2.3.0''
 * @param end The ending (or last) IP address in the range. for example: ''1.2.3.255''
 */
case class IpLimitRange (
  start: String,
  end: String
)

